src/components/Tabs/Tabs.scss

```diff
@@ -327,24 +326,88 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
 
   .Tab {
     padding: spacing(tight) spacing(extra-tight);
-
-    &:active .Title::before {
-      background: var(--p-action-primary-pressed);
+    &:hover .Title {
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
```

Comment:
I chose `rem(1px)` for the hover state - if it was at `rem(3px)` it would look the same as a selected tab.